### PR TITLE
Added announcement slider

### PR DESCRIPTION
**Pull request by devmode2022**
## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).

### CODE REVIEW

File: resources/js/pages/users/announcement/Landing.vue [hunk 1]
```diff
@@ -1,8 +1,22 @@
 <template>
     <div class="translate-container">
         <div class="row">
-            <div class="col-sm-12 col-md-8">
-                <announcement-slider></announcement-slider>
+            <div class="col-sm-12 col-md-8" v-if="announcements.length">
```
Comment (bestworks221) [inline resources/js/pages/users/announcement/Landing.vue:4]: we could just remove .length
`v-if="announcements"`

**Review by bestworks221 — CHANGES_REQUESTED** [review]
Please check comment

File: resources/js/pages/users/announcement/Landing.vue (hunk 1 above)
Comment (devmode2022) [reply resources/js/pages/users/announcement/Landing.vue:4]: hindi po pwede to Sir 

always true return nyan if announcements lang sya

**Review by bestworks221 — APPROVED** [review]
LGTM!

**Pull request merged**